src/object/srv_obj.c

```diff
@@ -925,6 +947,22 @@ obj_local_rw(crt_rpc_t *rpc, struct ds_cont_hdl *cont_hdl,
 		goto out;
```

Comment:
It's not from this patch, but it seems to me that cont_iv_prop_fetch() will be called in performance critical path (in obj_local_rw()), I'm afraid that'll be a performance killer when checksum is enabled.

I think we could cache the prop in the per-xstream ds_cont_child, and update the cached prop on IV refresh, so that read/write code path won't need to call into the heavy cont_iv_prop_fetch() anymore.